tests/1_host_configuration.sh

```diff
@@ -71,140 +71,200 @@ fi
 
 # 1.9
 check_1_9="1.9  - Audit Docker files and directories - /var/lib/docker"
-command -v auditctl >/dev/null 2>&1
-if [ $? -eq 0 ]; then
-  auditctl -l | grep /var/lib/docker >/dev/null 2>&1
+directory="/var/lib/docker"
```

Comment:
The container doesn't have access to this file, since we are not mounting `/var/lib` as a volume.